frontend/app/src/utils/color.ts

```diff
@@ -26,12 +26,16 @@ function randomInt(min: number, max: number): number {
 }
 
 function toHex(value: number): string {
-  return Math.round(value * 255)
+  return Math.round(value)
```

Comment:
What is the expectation for the value here? 0 to 1?

Reply:
it was 0 to 1, but I change it to 0 - 255, let me put some comment there.